libevm/VM.h

```diff
@@ -101,13 +103,18 @@ class VM: public VMFace
 	u256s m_stackSpace;
 	u256* m_stack;
 	
+	// space for return stack and pointer to data
+	std::vector<uint64_t> m_returnSpace;
+	uint64_t* m_return;
```

Comment:
There is only a default constructor.  I don't think you can inherit from this class without the derived constructor calling the inherited default constructor.  But I'll look at it after I have clean tests again.